build/gen_stub.php

```diff
@@ -2474,12 +2546,30 @@ function (Name $item) {
             $code .= $property->getDeclaration($allConstInfos);
         }
 
-        foreach ($this->attributes as $attribute) {
-            $code .= $attribute->generateCode("zend_add_class_attribute(class_entry", "class_$escapedName", $allConstInfos);
+        if (!empty($this->attributes)) {
+            if (!$php82MinimumCompatibility) {
+                $code .= "#if (PHP_VERSION_ID >= " . PHP_82_VERSION_ID . ")\n";
```

Comment:
```suggestion
                $code .= "\n#if (PHP_VERSION_ID >= " . PHP_82_VERSION_ID . ")";
```
attribute codegeneration inserts itself already a line before

Reply:
+1

Reply:
Thanks for the suggestion! I agree that it's much better now. :)